osu.Game/Overlays/Profile/Sections/PaginatedProfileSubsection.cs

```diff
@@ -153,9 +154,9 @@ protected virtual void OnItemsReceived(List<TModel> items)
         {
         }
 
-        protected abstract APIRequest<List<TModel>> CreateRequest(PaginationParameters pagination);
+        protected abstract APIRequest<List<TModel>> CreateRequest(APIUser user, PaginationParameters pagination);
```

Comment:
Previously, overrides of this method would access the user via the `User` bindable field. But the bindable was nullable all along, and there was no guarantee that `User.Value` was not null. Therefore this signature now includes a non-null `APIUser`, and the non-null guarantee is assured by the only caller checking `User.Value` for null prior to calling this method.